zhaquirks/xiaomi/__init__.py

```diff
@@ -438,6 +454,20 @@ def consumption_reported(self, value):
 
     async def read_attributes_raw(self, attributes, manufacturer=None):
```

Comment:
Just noticed something. This was fixed higher up and I neglected to use it. <https://github.com/dmulcahey/zha-device-handlers/blob/a918de150fa8cc646fa5d2547eafd5938ad6608a/zhaquirks/__init__.py#L33> if you switch `CustomCluster` to `LocalDataCluster` you should be able to drop this method all together.